tests/ui/asbytes-illegal.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 The Fuchsia Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#[derive(zerocopy::AsBytes)]
```

Comment:
Can you do `use zerocopy::AsBytes` at the top and then `#[derive(AsBytes)]` here (just to be consistent with other UI tests)? Same below.